### ISSUE TITLE
Multisig, as multi not supported

### ISSUE THREAD

**Issue by DanielHirsbrunner** [Zondax/ledger-kusama#95]
Steps to reproduce: 
Open Polkadot.js
Setup an Multisig Account (2 out of 3) with 3 Accounts on your ledger
Fund the account with some KSM (not to much, you want be able to withdraw it)
Create a new Transaction
Sign it with one of the accounts (this will be the first signature and therefore a mutlisig(approve_as_multi)
now Create the same operation again and sign it with an other account (Polkadot.js is smart enough to now create an multisig(asMulti) transaction)
Try to sign it

Result:
Polkadot.js shows that Ledger does not support this transaction

Expected Result:
The Transaction gets signed and broadcasted to the network

Additional information:
You can also create the approve as multi and as multi in the extrinsics, same outcome
I was able to do the exact same procedure with parity signer and it worked, therefore it must be a bug in the Ledger support

**Comment by jleni** [comment]
Could you indicate with app version are you using?
This feature has been added recently and I am not sure Ledger has actually published it yet

**Comment by jleni** [comment]
Also, as indicated here:
![image](https://user-images.githubusercontent.com/13698520/126278309-4284c6a2-760e-47eb-a88f-f079caeb7886.png)

This feature is available in XL version (Nano S devices) or Nano X.
The standard Nano S version (that requires less storage) does not support it.

**Comment by DanielHirsbrunner** [comment]
I use Ledger Nano X with the app Version 5.30.1. 
I see that you have newer Versions here in the repo, but according to Ledger Live there is no newer Version for me available.

**Comment by jleni** [comment]
Yes, updates and improvements have been already submitted to Ledger in this PR

https://github.com/LedgerHQ/app-kusama/pull/10

**Comment by jleni** [comment]
related PR https://github.com/LedgerHQ/app-kusama/pull/11
